chapters/operatorsandexpressions.tex

```diff
@@ -862,65 +845,54 @@ \subsubsection{spatialDistribution}\label{spatialdistribution}
   Integer m;
 algorithm
   if positiveVelocity then
-    out1:=interpolate(points, values, 1-(x-x0));
-    out0:=values[1]; // similar to in0 but avoiding algebraic loop
+    out1 := interpolate(points, values, 1-(x-x0));
+    out0 := values[1]; // similar to in0 but avoiding algebraic loop
   else
-    out0:=interpolate(points, values, 0-(x-x0));
-    out1:=values[end]; // similar to in1 but avoiding algebraic loop
+    out0 := interpolate(points, values, 0-(x-x0));
+    out1 := values[end]; // similar to in1 but avoiding algebraic loop
   end if;
   when <acceptedStep> then
     if x>x0 then
-      m:=size(points,1);
+      m := size(points, 1);
       while (if m>0 then points[m]+(x-x0)>=1 else false) then
-        m:=m-1;
+        m := m-1;
       end while;
-      values:=cat(1, {in0}, values[1:m], {interpolate(points, values,1-(x-x0))} );
-      points:=cat(1, {0}, points[1:m] .+ (x-x0), {1} );
+      values := cat(1, {in0}, values[1:m], {interpolate(points, values,1-(x-x0))});
+      points := cat(1, {0}, points[1:m] .+ (x-x0), {1});
     elseif x<x0 then
-      m:=1;
-      while (if m<size(points,1) then points[m]+(x-x0)<=0 else false) then
-        m:=m+1;
+      m := 1;
+      while (if m < size(points, 1) then points[m] + (x - x0) <= 0 else false) then
+        m := m+1;
       end while;
-      values:=cat(1, {interpolate(points, values, 0-(x-x0))},values[m:end],{in1});
-      points:=cat(1, {0}, points[m:end] .+ (x-x0), {1});
+      values := cat(1, {interpolate(points, values, 0 - (x - x0))}, values[m:end], {in1});
```

Comment:
Break line before `{interpolate` ?

Reply:
Fixed.  Resolving.  (Expecting any comments to be reported when looking at the use of `<and then>`.)